README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-    <img width=50% src="https://raw.githubusercontent.com/FeatureLabs/compose/master/docs/source/images/compose.png" alt="Compose" />
+    <img width=55% src="docs/source/images/compose.png" alt="Compose" />
```

Comment:
I'll need to switch back to an absolute URL path before merging to the base branch.